### DISCUSSION TITLE
有没有大佬帮忙解决一下啊

### DISCUSSION THREAD

**Discussion by li-chuanwei** [AlistGo/alist#3162] [Q&A]
<img width="618" alt="屏幕截图_20230127_154102" src="https://user-images.githubusercontent.com/115207784/215033841-80ba977f-126e-44a0-8b1c-c473223491b4.png">

**Comment by youtehub** [comment]
清除所有配置文件后重新安装试试

**Reply by li-chuanwei** [reply]
试过了,还是不行第一步就搞不定😅
